File: Components/RoomSelector.py
import flask


app = flask.Flask(__name__)

@app.route("/selectRoom", methods=["POST", "GET"])
def selectRoom():
    if flask.request.method == "POST":
        return "Room Selected is {} and guests {}".format(flask.request.form['roomTypesList'],flask.request.form['noOfGuests'])
    else:
        return """
        <form method="POST">
        <select id="roomTypesList" name="roomTypesList">
            <option value="empty">Select Room Type</option>
            <option value="ac_deluxe">AC Deluxe</option>
            <option value="non_ac_deluxe">Non AC Deluxe</option>
            <option value="Dorm">Dormitory</option>
            <option value="suite">Executive Suite</option>
            <option value="presidentSuite">Presidential Suite</option>
            <option value="cabana">Cabana</option>
            <option value="villa">Villa</option>
            <option value="single">Single</option>
            <option value="double">Double</option>
        </select>
        <select id="noOfGuests" name="noOfGuests">
            <option value="empty">Select number of Guests</option>
            <option value="1">1</option>
            <option value="2">2</option>
            <option value="3">3</option>
            <option value="4">4</option>
            <option value="5">5</option>
            <option value="6">6</option>
            <option value="7">7</option>
            <option value="8">8</option>
            <option value="9">9</option>
            <option value="10">10</option>
        </select>
        <input type="submit"></input>
</form>
"""

File: Components/RoomSelector.java
import javax.swing.*;

public class RoomSelector {

	public static void main(String[] args) {
		JFrame frame = new JFrame("Select Room");
		frame.setSize(300, 150);
		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);

		JPanel panel = new JPanel();
		frame.add(panel);
		placeComponents(panel);

		frame.setVisible(true);
	}

	private static void placeComponents(JPanel panel) {

		panel.setLayout(null);

		JLabel roomTypeLabel = new JLabel("Room Type");
		roomTypeLabel.setBounds(10, 10, 80, 25);
		panel.add(roomTypeLabel);

        String[] roomTypes={
            "AC Deluxe",
            "Non AC Deluxe",
            "Dormitory",
            "Executive Suite",
            "Presidential Suite",
            "Cabana",
            "Villa",
            "Single",
            "Double"
        };
		JComboBox roomTypeComboBox = new JComboBox(roomTypes);
		roomTypeComboBox.setBounds(100, 10, 160, 25);
		panel.add(roomTypeComboBox);

		JLabel numGuestsLabel = new JLabel("Number of Guests");
		numGuestsLabel.setBounds(10, 40, 150, 25);
		panel.add(numGuestsLabel);

		String[] numbers={"1","2","3","4","5","6","7","8","9","10"};
        JComboBox numberOfGuestsComboBox = new JComboBox(numbers);
		numberOfGuestsComboBox.setBounds(160, 40, 100, 25);
		panel.add(numberOfGuestsComboBox);

		JButton submitButton = new JButton("submit");
		submitButton.setBounds(10, 80, 150, 25);
		panel.add(submitButton);
		
	}

}